misc/testerina/modules/testerina-runtime/src/main/java/org/ballerinalang/test/runtime/util/TesterinaUtils.java

```diff
@@ -292,14 +292,16 @@ private static void printStackElement(StringBuilder sb, StackTraceElement stackT
         String pkgName = Utils.decodeIdentifier(stackTraceElement.getClassName());
         String fileName = stackTraceElement.getFileName();
 
-        // clean file name from pkgName since we print the file name after the method name.
```

Comment:
We can improve this code by just adding the following null check to the fileName before L295:

```java
if (fileName != null) {
  fileName = "unknown-source"
}
```

Reply:
Changed as suggested